glide.lock

```diff
@@ -0,0 +1,76 @@
+hash: 5bee3b244d1e24f880d598b0b4e7b8868cfaad140e1757b3a0c4b37f0259a96e
+updated: 2016-11-28T15:55:02.79276474+01:00
+imports:
+- name: github.com/intelsdi-x/snap
+  version: 891c09fd7e93aaf7cc0fb976fb91407901fd3843
```

Comment:
This indicates Snap's commit from `Apr 28 Thu Apr 28 18:42:14 2016`. Could you make an update and use newer Snap's commit?